buildtools/pipenv/pipenv.go

```diff
@@ -0,0 +1,148 @@
+package pipenv
+
+import (
+	"encoding/json"
+
+	"github.com/fossas/fossa-cli/errors"
+	"github.com/fossas/fossa-cli/exec"
+	"github.com/fossas/fossa-cli/graph"
+	"github.com/fossas/fossa-cli/pkg"
+)
+
+// Pipenv defines the interface for all pipenv tool implementation.
+type Pipenv interface {
+	Deps() (graph.Deps, error)
+}
+
+// PipenvCmd implements Pipenv by parsing command output.
+type PipenvCmd struct {
+	Dir   string
+	Graph func(dir string) (string, error)
+}
+
+// dependency is used to unmarshal the output from `pipenv graph --json-tree`
+// and store an object representing an imported dependency.
```

Comment:
[nit] This second line doesn't add much extra value to the comment. Also, I would argue that struct values are not objects in the classical OOP sense -- they aren't capable of dynamic dispatch or subtyping.